.github/workflows/ci.yml

```diff
@@ -15,7 +15,6 @@ jobs:
         gem install rubocop rubocop-performance --no-document
         rubocop --require rubocop-performance \
                 --format progress \
-                --out ~/test-results/linting/rubocop.xml
```

Comment:
Oh, the line continuation line on the line before should also be removed.

Reply:
It still _worked_, though.

Reply:
Perhaps we can reduce all the way down to `rubocop --require rubocop-performance`